bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -226,21 +233,30 @@ public void setNumberOfEvents(int numberOfEvents) {
 
     public void configureInstallationInstance(BigDecimal installationId)
             throws ExecutionException, InterruptedException, TimeoutException {
-        csrf = getCsrfToken(installationId);
-        logger.debug("Got CSRF: {}", csrf);
-        // Set installation
         String url = SET_INSTALLATION + installationId;
         httpClient.GET(url);
     }
 
-    public @Nullable String getCsrfToken(BigDecimal installationId)
-            throws ExecutionException, InterruptedException, TimeoutException {
+    public @Nullable String getCsrfToken() throws ExecutionException, InterruptedException, TimeoutException {
         String html = null;
-        String url = SETTINGS + installationId;
+        CookieStore originalCookieStore = httpClient.getCookieStore();
+        httpClient.setCookieStore(new HttpCookieStore.Empty());
+
+        ContentResponse response = httpClient.newRequest(LOGIN).method(HttpMethod.GET)
+                .header(HttpHeader.HOST, "mypages.verisure.com").header(HttpHeader.CONNECTION, "keep-alive")
+                .header(HttpHeader.ACCEPT_LANGUAGE, "sv-SE,sv;q=0.9,en-US;q=0.8,en;q=0.7")
+                .header(HttpHeader.ACCEPT_ENCODING, "gzip, deflate, br, zstd").header(HttpHeader.ACCEPT, "*/*").send();
```

Comment:
The web application, that I have reverse engineered, explicitly sets these request headers.
I have not tried to exclude them one by one, since they are set by the web app.
After doing tests it seems like that they are not necessary to set.